### PR TITLE
[#3680] docs(trino-connector): Update Trino connector doc for using Trino dynamic catalog

### PR DESCRIPTION
**Pull request by diqiu50**
### What changes were proposed in this pull request?

 Update Trino connector doc for using Trino dynamic catalog

### Why are the changes needed?

Fix: #3680 

### Does this PR introduce _any_ user-facing change?

Update docs

### How was this patch tested?

No

### CODE REVIEW

File: docs/trino-connector/development.md [hunk 1]
```diff
@@ -77,142 +72,75 @@ then you can see the `gravitino-trino-connecor` source files and directories in
...
+            <version>2.0.9</version>
         </dependency>
 
         <dependency>
-            <groupId>org.openjdk.jol</groupId>
-            <artifactId>jol-core</artifactId>
-            <scope>provided</scope>
+            <groupId>com.datastrato.gravitino</groupId>
+            <artifactId>gravitino-client-java-runtime</artifactId>
+            <version>0.6.0-SNAPSHOT</version>
```
Comment (yuqi1129) [inline docs/trino-connector/development.md:133]: `0.6.0-SNAPSHOT` has not been release, you may need to change to 0.5.0 and add some comment like 
```
You can switch to the snapshot version as you like,  for example, if you want to latest main branch, you can execute the following command the install Gravitino `gravitino-client-java-runtime` jar locally.
./gradlew publishToMavenLocal

```

Comment (yuqi1129) [reply docs/trino-connector/development.md:133]: Additionally, the artifact ID is incorrect. This is the version in the Maven central repository.

<dependency>
    <groupId>com.datastrato.gravitino</groupId>
    <artifactId>client-java-runtime</artifactId>
    <version>0.5.0</version>
</dependency>

image

File: docs/trino-connector/configuration.md [hunk 2]
```diff
@@ -6,9 +6,13 @@ license: "Copyright 2023 Datastrato Pvt Ltd.
...
+| gravitino.simplify-catalog-names | boolean | true                        | The `gravitino.simplify-catalog-names` setting omits the metalake prefix from catalog names when set to true.                                                                                                                                                                                                       | NO       | 0.5.0         |
+| trino.jdbc.uri                   | string  | jdbc:trino://localhost:8080 | The jdbc uri of current Trino server.                                                                                                                                                                                                                                                                               | NO       | 0.5.1         |
```
[the remark this answers is not in the document]
Comment (diqiu50) [reply docs/trino-connector/configuration.md:15]: The Trino to install the Gravitino connector

File: docs/trino-connector/development.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ To develop the Gravitino connector locally, you need to do the following steps:
 
 ### IDEA
 
-1. Clone the Trino repository from the [GitHub](https://github.com/trinodb/trino) repository. We advise you to use the release version 426 or 435. 
+1. Clone the Trino repository from the [GitHub](https://github.com/trinodb/trino) repository. We advise you to use the release version 435.
```
[the remark this answers is not in the document]
Comment (diqiu50) [reply docs/trino-connector/development.md:40]: It's supported.

File: docs/trino-connector/installation.md [hunk 4]
```diff
@@ -64,6 +76,20 @@ cd /lib/trino/plugin
 
 Now you can see the Gravitino connector directory in the plugin directory.
 
+### Configuring the Trino
+
+You can find the Trino configuration file in the `/etc/trino/config.properties`. You need changed the file like this:
+
+```test
```
Comment (jerryshao) [inline docs/trino-connector/installation.md:83]: text not test.

**Comment by jerryshao** [comment]
@yuqi1129 please help to review this.

File: docs/trino-connector/development.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ To develop the Gravitino connector locally, you need to do the following steps:
 
 ### IDEA
 
-1. Clone the Trino repository from the [GitHub](https://github.com/trinodb/trino) repository. We advise you to use the release version 426 or 435. 
+1. Clone the Trino repository from the [GitHub](https://github.com/trinodb/trino) repository. You need to use the release version at least Trino-435.
```
Comment (yuqi1129) [inline docs/trino-connector/development.md:40]: The released version Trino-435 is the least version that Gravitino supports.

**Review by yuqi1129 — APPROVED** [review]

**Pull request merged**